### PR TITLE
Feature/command migration locale migration

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
All changes regarding the locale code renaming. Additionally English_UK is now supported. Portuguese is no longer supported but the Brazilian,

### CODE REVIEW

File: src/main/resources/locale_es_ES.properties [hunk 1]
```diff
@@ -258,45 +257,44 @@ command.scan.start.message.queueFull=Se están ejecutando demasiados escaneos ac
...
 command.thankwords.list.description=Lista de todos los patrones
 command.thankwords.list.message.list=Los siguientes patrones están activos:
 command.thankwords.loaddefault.description=Cargar palabras de agradecimiento por defecto para un idioma.
-command.thankwords.loaddefault.message.added=Añadir las siguientes palabras de agradecimiento:
 command.thankwords.loaddefault.language.description=El idioma para el que se deben cargar las palabras de agradecimiento.
+command.thankwords.loaddefault.message.added=Añadir las siguientes palabras de agradecimiento:
 command.thankwords.loaddefault.message.available=Idiomas disponibles:
 command.thankwords.remove.description=Eliminar un patrón
-command.thankwords.remove.pattern.description=El patrón regex a eliminar.
+command.thankwords.remove.message.patternnotnound=El patrón no existe.
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:290]: Looks like there is a typo in the key

File: src/main/resources/locale_pt_BR.properties [hunk 2]
```diff
@@ -0,0 +1,361 @@
+command.abuseprotection.cooldown.description=Obter ou definir o arrefecimento da reputação.
+command.abuseprotection.cooldown.message.get=Você pode enviar reputação para um usuário a cada %MINUTES% minutos.
+command.abuseprotection.cooldown.message.set=Cooldown definido para %MINUTES% minutos.
+command.abuseprotection.cooldown.minutes.description=Quantidade de minutos que os usuários não podem agradecer uns aos outros novamente.
+command.abuseprotection.description=Receba as configurações atuais de proteção contra abuso.
+command.abuseprotection.donorcontext.description=Obtenha ou defina se o doador precisa ou não de mensagens recentes.
+command.abuseprotection.donorcontext.message.false=O doador não precisa de mensagens recentes no canal.
+command.abuseprotection.donorcontext.message.true=O doador deve ter mensagens recentes no canal.
+command.abuseprotection.donorcontext.state.description=Defina como "verdadeiro" para ativar as verificações de contexto do doador.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_BR.properties:9]: I think the quoted text should be `true`?
```suggestion
command.abuseprotection.donorcontext.state.description=Defina como "true" para ativar as verificações de contexto do doador.
```

File: src/main/resources/locale_pt_BR.properties [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+command.roles.removedonor.description=Remover um papel de doador.
+command.roles.removedonor.role.description=O papel do doador a ser removido.
+command.roles.removereceiver.description=Remover uma função do receptor
+command.roles.removereceiver.message.remove=Removido o papel do receptor %ROLE%
+command.roles.removereceiver.role.description=O papel do receptor a ser removido.
+command.roles.removeronor.message.remove=O papel do doador removido %ROLE%
+command.roles.stackroles.description=Permitir o empilhamento de papéis
+command.roles.stackroles.message.notStacked=O usuário só terá o papel mais próximo de sua reputação.
+command.roles.stackroles.message.stacked=Os papéis serão empilhados. Os usuários terão todos os papéis com menos do que a sua reputação.
+command.roles.stackroles.stack.description=Definido como "verdadeiro" para empilhar.
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_BR.properties:243]:
```suggestion
command.roles.stackroles.stack.description=Definido como "true" para empilhar.
```

**Review by Taucher2003 — COMMENTED** [review]
en_US and en_GB are completely identical, what is the value of having both?

**Comment by RainbowDashLabs** [comment]
@Taucher2003 I removed the gb file for now, but maybe some day we will have actual differences. But if this ever happens we will just readd it again.

I also noticed that we need to rename a couple of locale codes as well and added them to the migration script

**Pull request merged**